charts/victoria-metrics-operator/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+weight: 9
```

Comment:
It has the same weight as the previous K8s Stack.

Reply:
updated it as well